vita3k/ngs/src/modules/atrac9.cpp

```diff
@@ -182,10 +182,10 @@ bool Atrac9Module::decode_more_data(KernelState &kern, const MemState &mem, cons
             decoded_size -= skipped_samples;
         }
 
-        const uint32_t samples_left_after = (frame_bytes_gotten / superframe_size - 1) * samples_per_superframe;
+        const uint32_t samples_left_after = ((frame_bytes_gotten / superframe_size) - 1) * samples_per_superframe;
```

Comment:
Revert the unnecessary brackets

Reply:
I like it personaly get more clear view ^^